app.js

```diff
@@ -84,14 +57,8 @@ var server = http.createServer(function(request, response) {
     var tranny = query.indexOf('trans') === -1 ? ' -trans' : '';
     var shemale = query.indexOf('shemale') === -1 ? ' -shemale' : '';
 
-    if (gay || tranny || shemale) {
```

Comment:
It's special check that allows "uncommon" videos in results if you write one special words. It's better to return this code back.